pages/common/mix.md

```diff
@@ -0,0 +1,27 @@
+# mix
+
+> Mix is a build tool that provides tasks for creating, compiling, and testing Elixir projects, managing its dependencies, and more.
+
+- List all mix commands:
```

Comment:
This should be the last example. Think of the natural order of commands that a user is expected to use. First, a user will want to run a file and see if it works or not. Then move on to creating projects and compiling them. And lastly, they would want to see all commands.